src/backend/services/auth/jwt.py

```diff
@@ -39,7 +39,7 @@ def create_and_encode_jwt(self, user: dict) -> str:
         payload = {
             "iss": self.ISSUER,
             "iat": now,
-            "exp": now + datetime.timedelta(months=self.EXPIRY_MONTHS),
+            "exp": now + datetime.timedelta(days=self.EXPIRY_DAYS),
```

Comment:
Unrelated change, but this was breaking tests with the error `months is not a valid keyword`